pkg/devfile/adapters/kubernetes/adapter.go

```diff
@@ -28,6 +29,14 @@ func New(adapterContext common.AdapterContext, client kclient.Client) Adapter {
 	}
 }
 
+func (k Adapter) ExecCMDInContainer(compInfo common.ComponentInfo, cmd []string, stdout io.Writer, stderr io.Writer, stdin io.Reader, tty bool) error {
```

Comment:
Please mention comment for what does this function do

Reply:
This method was removed.